packages/pure/src/device/device.types.ts

```diff
@@ -28,8 +28,23 @@ export enum McUsbFileType {
   mp3 = 0x3009,
   flac = 0xb906,
 }
+export interface McSerialPortDevice {
+  path: string
+  deviceType: DeviceType
+  connect(): Promise<Response>
+  disconnect(): Promise<Response>
+  request(config: RequestConfig<any>): Promise<Response<any>>
+  on(eventName: DeviceEventName, listener: () => void): void
+  off(eventName: DeviceEventName, listener: () => void): void
+}
+
+export interface McUsbDevice {
+  getFiles(): Promise<McUsbFile[]>
+}
 
 export type CreateDeviceStrategy = (
+  baseMcSerialPortDevice: McSerialPortDevice,
+  baseMcUscDevice: McUsbDevice,
   path: string,
   deviceType: DeviceType
```

Comment:
```suggestion
  baseMcUscDevice: McUsbDevice
```

Reply:
Removed in [8045a34](https://github.com/mudita/mudita-center/pull/681/commits/8045a34fa0741889e6bbbacbe34816ca989e3107